chrome/content/zotero/bibliography.js

```diff
@@ -195,10 +212,11 @@ var Zotero_File_Interface_Bibliography = new function() {
 		
 		// Change label to "Citation" or "Note" depending on style class
 		if(document.getElementById("citations")) {
+			var label = "";
```

Comment:
By the way, is there a preferred way to deal with variables that are first needed in if/else constructs?

My copy of JSHint doesn't like the construct

```
if (...) {
  var foo = bar;
} else {
  var foo = baz;
}
```

since it thinks repeated variable declarations are bad. Should I define "foo" before the conditional statement, as I did here, or just ignore the warning?